src/workflows-app/SubmissionDetails.test.js

```diff
@@ -312,14 +210,14 @@ describe('Submission Details page', () => {
     const cellsFromDataRow1 = within(rows[1]).queryAllByRole('cell');
     expect(cellsFromDataRow1.length).toBe(3);
     await within(cellsFromDataRow1[0]).findByText('FOO1');
-    await within(cellsFromDataRow1[1]).findByText('Succeeded');
-    await within(cellsFromDataRow1[2]).findByText('37 seconds');
+    within(cellsFromDataRow1[1]).getByText('Succeeded');
+    within(cellsFromDataRow1[2]).getByText('37 seconds');
 
     const cellsFromDataRow2 = within(rows[2]).queryAllByRole('cell');
     expect(cellsFromDataRow2.length).toBe(3);
     await within(cellsFromDataRow2[0]).findByText('FOO2');
-    await within(cellsFromDataRow2[1]).findByText('Failed');
-    await within(cellsFromDataRow2[2]).findByText('52 minutes 10 seconds');
+    within(cellsFromDataRow2[1]).getByText('Failed');
+    within(cellsFromDataRow2[2]).getByText('52 minutes 10 seconds');
 
     // Act - click on sort button on Duration column to sort by descending order
     user.click(await within(headers[2]).findByRole('button'));
```

Comment:
`userEvent.click` returns a Promise and should be awaited.
https://testing-library.com/docs/user-event/convenience#click

This might explain the need for `findBy...` queries below.